EventFilter/HcalRawToDigi/interface/HcalDataFrameFilter.h

```diff
@@ -28,14 +29,21 @@ class HcalDataFrameFilter {
   HcalCalibDigiCollection filter(const HcalCalibDigiCollection& incol, HcalUnpackerReport& r);
   /// filter ZDC data frames
   ZDCDigiCollection filter(const ZDCDigiCollection& incol, HcalUnpackerReport& r);
+  /// filter QIE10 data frames
+  QIE10DigiCollection filter(const QIE10DigiCollection& incol, HcalUnpackerReport& r);
+  /// filter QIE11 data frames
+  QIE11DigiCollection filter(const QIE11DigiCollection& incol, HcalUnpackerReport& r);
   /// whether any filters are on
   bool active() const;
+  /// get conditions
+  void setConditions(const HcalDbService* conditions);
 private:
   bool requireCapid_;
   bool requireDVER_;
   bool energyFilter_;
   int firstSample_, lastSample_;
   double minimumAmplitude_;
+  const HcalDbService* conditions_;
```

Comment:
since I was looking I'll comment early - I find this dependency very odd.... would naively belong as a es product.. (maybe) - thats how it seems to be used elsewhere..